Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:2.7.13
+
+ENV DEBIAN_FRONTEND noninteractive
+
+WORKDIR /app
+
+COPY requirements.txt /tmp/requirements.txt
+RUN pip install --upgrade pip
+RUN pip install -r /tmp/requirements.txt
+
+RUN apt-get update \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
Combine these into a single `RUN` or something... they affect image size (unless we switch to multi-stage builds).